library/RevAPI/AbstractOrderSubmission.php

```diff
@@ -0,0 +1,159 @@
+<?php
+
+namespace RevAPI;
+
+abstract class AbstractOrderSubmission {
+    
+    /**
+     * @var Rev
+     */
+    protected $rev;
+    
+    protected $client_ref = null;
+    
+    protected $comment = null;
+    
+    protected $priority = 'Normal';
+    
+    protected $inputs = array();
+
+    protected $notification = array();
+    
+    const PRIORITY_TIME_INSENSITIVE = 'TimeInsensitive';
```

Comment:
As a coding standard, I like constants to be at the top of the class (above members).